### PR TITLE
Feature/22 Configure Spring Boot

### PR DESCRIPTION
**Pull request by olena-syrotenko**

### CODE REVIEW

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
```
Comment (rdovgan) [inline .mvn/wrapper/maven-wrapper.properties:1]: @olena-syrotenko please add `.mvn/*` to .gitignore file

File: mvnw.cmd [hunk 2]
```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```
Comment (rdovgan) [inline mvnw.cmd:1]: @olena-syrotenko this file also should be ignored

File: src/test/java/team/asd/InternApplicationTest.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package team.asd;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class InternApplicationTests {
```
Comment (rdovgan) [inline src/test/java/team/asd/InternApplicationTest.java:7]: @olena-syrotenko please rename class according to filename and add an empty line at the end of this file

File: .mvn/wrapper/maven-wrapper.properties (hunk 1 above)
Comment (olena-syrotenko) [reply .mvn/wrapper/maven-wrapper.properties:1]: done

File: mvnw.cmd (hunk 2 above)
Comment (olena-syrotenko) [reply mvnw.cmd:1]: done

File: src/test/java/team/asd/InternApplicationTest.java (hunk 3 above)
Comment (olena-syrotenko) [reply src/test/java/team/asd/InternApplicationTest.java]: done

**Review by vasylnikorych — APPROVED** [review]

**Pull request merged**